<?xml version="1.0"?>
<launch>

		<arg name="datapath"	default="/home/robot/Desktop/ALuque/mpc_ros/src/mpc_node/data/"/>

		<arg name="NTraj"		  default="10"/>
		<arg name="nSOM"		  default="4"/>
		<arg name="nCOM"		  default="4"/>
		<arg name="Hp"			  default="25"/>
		<arg name="Ts"			  default="0.02"/>
		<arg name="Wv"				default="0.20"/>

    <node name="mpc_cl_node"
    			pkg="mpc_pkg"
    			type="mpc_cl_node"
    			output="screen">
    			<param name="datapath" value="$(arg datapath)"/>
    			<param name="NTraj" value="$(arg NTraj)"/>
    			<param name="nSOM" value="$(arg nSOM)"/>
    			<param name="nCOM" value="$(arg nCOM)"/>
    			<param name="Hp" value="$(arg Hp)"/>
    			<param name="Ts" value="$(arg Ts)"/>
    			<param name="Wv" value="$(arg Wv)"/>
    </node>

</launch>

